libs/efivar/files/efivar-guids.h

```diff
@@ -0,0 +1,80 @@
+#ifndef EFIVAR_GUIDS_H
```

Comment:
where do these files come from? can't them be upstreamed?

Reply:
normal linux efivar headers....

Reply:
They came from somewhere - but I cannot find them now afterwards from anywhere. And I though that they were needed for a reason.. Maybe they were for older version of efivar, hard to say. I removed them and patches equipping them.